tools/rewriter/GenCallAsm.cpp

```diff
@@ -1207,6 +1217,36 @@ std::string emit_asm_wrapper(
 
   emit_prologue(aw, caller_pkey, target_pkey, arch);
 
+  if (trace_target_pkey.has_value()) {
+    if (arch == Arch::X86) {
+      add_asm_line(aw, ".ifdef IA2_TRACE_EXIT");
```

Comment:
I don't think this works. `.ifdef` expects an ELF symbol (e.g. global var or function) named `IA2_TRACE_EXIT` but I think we only define a preprocessor macro named `IA2_TRACE_EXIT`